mqtt-kafka-connect/README.md

```diff
@@ -0,0 +1,177 @@
+# kafnus
+# csv-source-custom-postgres-sink-kafka-connect
+
+Demo of how to stream CSV file to Kafka and use a custom sink Postgres connector. 
+
+## Data
+
+The data used here is a MP3 playlist metadata. These CSV files are in `data` folder.
+
+## Components
+
+The following technologies are used through Docker containers:
+* Kafka, the streaming platform
+* Kafka's schema registry, needed to use the Avro data format
+* Kafka Connect, to orchestrate csv file source and Postgres sink.
+* Postgres, pulled from [Debezium](https://debezium.io/), tailored for use with Connect
+* [Java 11+](https://openjdk.java.net)
+* [Apache Maven](https://maven.apache.org)
+
+The containers are pulled directly from official Docker Hub images.
+The Connect image used here needs some additional packages, some of them from `libs` folder, so it must be built from the 
+included Dockerfile.
+
+## ⚙️ Building components
+
+### Build the Kafka Connect image
+```
+docker build -t connector -f connector.Dockerfile .
+```
+
+### Create volume for Postgres
+```
+docker volume create postgresdata
+```
+
+### Build the custom Postgres sink Kafka Connect
```

Comment:
I guess all related with kafka connector for postgres should be the same that https://github.com/telefonicaid/kafnus/pull/10 and https://github.com/telefonicaid/kafnus/pull/9, or at least should be addressed at the same point.